examples/signature/recurring_swap_deploy.py

```diff
@@ -14,6 +14,17 @@
 
 from recurring_swap import recurring_swap
 
+# user declared algod connection parameters. Node must have EnableDeveloperAPI set to true in its config
```

Comment:
I went down the wrong rabbit hole with this script thinking this was using templates. By the time I had realized these weren't the "real" templates example, I had already fixed this broken script. Might as well keep it fixed.